packages/block-library/src/cover/save.js

```diff
@@ -144,6 +148,16 @@ export default function save( { attributes } ) {
 					data-object-position={ objectPosition }
 				/>
 			) }
+			{ isEmbedVideoBackground && embedSrc && (
+				<div className="wp-block-cover__video-background wp-block-cover__embed-background">
+					<iframe
```

Comment:
It's interesting that you rendered the iframe manually here. Shouldn't we rely on the WP embed behavior instead? (like just rendering the **url** here like the embed block does and let the server side hooks transform it properly).